164544
201603
{'11': {'Retries': 2, 'WallDurations': [10.0, 1209.0, 9.0, 1415.0, 9.0, 1207.0], 'StartTimes': [1494241329.0, 1494242858.0, 1494244527.0], 'SubmitTimes': [1494240767.0, 1494242607.0, 1494244402.0], 'JobIds': ['13830172.0', '13830788.0', '13831871.0'], 'EndTimes': [1494242538.0, 1494244273.0, 1494245734.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [291844, 295064, 293592], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '10': {'Retries': 2, 'WallDurations': [10.0, 1207.0, 10.0, 1208.0, 9.0, 1232.0], 'StartTimes': [1494240958.0, 1494242373.0, 1494244251.0], 'SubmitTimes': [1494240766.0, 1494242233.0, 1494243708.0], 'JobIds': ['13830169.0', '13830754.0', '13831820.0'], 'TotalUserCpuTimeHistory': [3, 3, 4], 'EndTimes': [1494242165.0, 1494243581.0, 1494245483.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [287280, 24608, 292212], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, 'DagStatus': {'Timestamp': 1494252945L, 'NodesTotal': 11L, 'DagStatus': 6L}, '1': {'Retries': 2, 'WallDurations': [9.0, 1207.0, 1974.0, 8.0, 7289.0], 'StartTimes': [1494241796.0, 1494243164.0, 1494245648.0], 'SubmitTimes': [1494240766.0, 1494243072.0, 1494245268.0], 'JobIds': ['13830168.0', '13830850.0', '13832320.0'], 'EndTimes': [1494243003.0, 1494245138.0, 1494252937.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [293596, 294312, 154908], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '3': {'Retries': 2, 'WallDurations': [9.0, 1208.0, 9.0, 1955.0, 9.0, 1207.0], 'StartTimes': [1494241632.0, 1494243165.0, 1494245503.0], 'SubmitTimes': [1494240766.0, 1494242911.0, 1494245247.0], 'JobIds': ['13830166.0', '13830830.0', '13832319.0'], 'TotalUserCpuTimeHistory': [3, 3, 3], 'EndTimes': [1494242840.0, 1494245120.0, 1494246710.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [292212, 295432, 292328], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '2': {'Retries': 2, 'WallDurations': [9.0, 1207.0, 9.0, 1209.0, 9.0, 1207.0], 'StartTimes': [1494241941.0, 1494243302.0, 1494244801.0], 'SubmitTimes': [1494240767.0, 1494243217.0, 1494244639.0], 'JobIds': ['13830173.0', '13830861.0', '13831901.0'], 'TotalUserCpuTimeHistory': [3, 3, 3], 'EndTimes': [1494243148.0, 1494244511.0, 1494246008.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [294388, 294616, 294664], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '5': {'Retries': 2, 'WallDurations': [9.0, 1207.0, 8.0, 1207.0, 9.0, 1335.0], 'StartTimes': [1494242117.0, 1494243597.0, 1494245137.0], 'SubmitTimes': [1494240767.0, 1494243395.0, 1494244930.0], 'JobIds': ['13830174.0', '13831434.0', '13831926.0'], 'EndTimes': [1494243324.0, 1494244804.0, 1494246472.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [293900, 294752, 294088], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '4': {'Retries': 2, 'WallDurations': [9.0, 1207.0, 9.0, 2152.0, 9.0, 7210.0], 'StartTimes': [1494241631.0, 1494243020.0, 1494245721.0], 'SubmitTimes': [1494240766.0, 1494242905.0, 1494245299.0], 'JobIds': ['13830167.0', '13830824.0', '13832324.0'], 'EndTimes': [1494242838.0, 1494245172.0, 1494252931.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [295860, 296944, 294324], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '7': {'Retries': 2, 'WallDurations': [8.0, 1206.0, 9.0, 2178.0, 10.0, 1207.0], 'StartTimes': [1494241154.0, 1494242666.0, 1494245092.0], 'SubmitTimes': [1494240766.0, 1494242429.0, 1494244971.0], 'JobIds': ['13830164.0', '13830775.0', '13831932.0'], 'EndTimes': [1494242360.0, 1494244844.0, 1494246299.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [293752, 293632, 283472], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '6': {'Retries': 2, 'WallDurations': [9.0, 1206.0, 2586.0, 9.0, 1211.0], 'StartTimes': [1494241152.0, 1494242554.0, 1494245370.0], 'SubmitTimes': [1494240766.0, 1494242424.0, 1494245268.0], 'JobIds': ['13830171.0', '13830774.0', '13832321.0'], 'TotalUserCpuTimeHistory': [3, 3, 3], 'EndTimes': [1494242358.0, 1494245140.0, 1494246581.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [294556, 294680, 24372], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '9': {'Retries': 2, 'WallDurations': [9.0, 1207.0, 9.0, 1896.0, 8.0, 1207.0], 'StartTimes': [1494240958.0, 1494242377.0, 1494244496.0], 'SubmitTimes': [1494240766.0, 1494242234.0, 1494244402.0], 'JobIds': ['13830165.0', '13830755.0', '13831870.0'], 'EndTimes': [1494242165.0, 1494244273.0, 1494245703.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [290644, 296940, 291824], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213llrxrd-redir.in2p3.fr,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-redic.pi.infn.it,+1213llrxrd-redir.in2p3.fr,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd.ba.infn.it,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}, '8': {'Retries': 2, 'WallDurations': [10.0, 1207.0, 9.0, 1424.0, 8.0, 1207.0], 'StartTimes': [1494241154.0, 1494242851.0, 1494244632.0], 'SubmitTimes': [1494240766.0, 1494242429.0, 1494244407.0], 'JobIds': ['13830170.0', '13830776.0', '13831872.0'], 'EndTimes': [1494242361.0, 1494244275.0, 1494245839.0], 'Restarts': 0, 'RecordedSite': True, 'State': 'failed', 'ResidentSetSize': [293480, 294388, 294644], 'Error': [8028, "CmsRunFailure\nCMSSW error message follows.\nFatal Exception\nAn exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094\n", {'type': 'Fatal Exception', 'details': "An exception of category 'FallbackFileOpenError' occurred while\n   [0] Constructing the EventProcessor\n   [1] Constructing input source of type PoolSource\n   [2] Calling RootFileSequenceBase::initTheFile()\n   [3] Calling StorageFactory::open()\n   [4] Calling XrdFile::open()\nException Message:\nFailed to open the file 'root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root'\n   Additional Info:\n      [a] XrdCl::File::Open(name='root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [b] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd.ba.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [c] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [d] Disabled source: cms-xrd-transit.cern.ch:1094\n      [e] Input file root://se01.grid.nchc.org.tw//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root could not be opened.\nFallback Input file root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root also could not be opened.\nOriginal exception info is above; fallback exception info is below.\n      [f] XrdCl::File::Open(name='root://xrootd-cms.infn.it//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root', flags=0x10, permissions=0660) => error '[ERROR] Server responded with an error: [3011] No servers are available to read the file.\n' (errno=3011, code=400). No additional data servers were found.\n      [g] Last URL tried: root://cms-xrd-transit.cern.ch:1094//store/mc/RunIIWinter15wmLHE/ZJ_ZToMuMu_powheg_minlo_8TeV_CT14/LHE/MCRUN2_71_V1-v1/110000/58B398B2-E3F5-E611-976A-001A649D5019.root?tried=+1098xrdcmsglobal011098xrdcmsglobal02.cern.ch,+1213xrootd.ba.infn.it,+1213xrootd-redic.pi.infn.it,+1213xrootd-cms-redir-01.cr.cnaf.infn.it,+1213llrxrd-redir.in2p3.fr,+se01.grid.nchc.o.1213xrootd-redic.pi1213xrootd-cms-redir-01.cr.cnaf1213llrxrd-redir.in2p3.fr1213xrootd.ba.infn.it,\n      [h] Problematic data server: cms-xrd-transit.cern.ch:1094\n      [i] Disabled source: cms-xrd-transit.cern.ch:1094", 'exitCode': '8028'}], 'TotalUserCpuTimeHistory': [3, 3, 3], 'SiteHistory': ['Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC', 'Unknown', 'T2_TW_NCHC'], 'TotalSysCpuTimeHistory': [0, 0, 0]}}
{(13831872L, 0L): '8', (13830173L, 0L): '2', (13830168L, 0L): '1', (13832319L, 0L): '3', (13830788L, 0L): '11', (13830174L, 0L): '5', (13830824L, 0L): '4', (13830164L, 0L): '7', (13830850L, 0L): '1', (13830169L, 0L): '10', (13830830L, 0L): '3', (13831434L, 0L): '5', (13832324L, 0L): '4', (13830774L, 0L): '6', (13830861L, 0L): '2', (13832320L, 0L): '1', (13831870L, 0L): '9', (13830165L, 0L): '9', (13830170L, 0L): '8', (13831901L, 0L): '2', (13830754L, 0L): '10', (13830775L, 0L): '7', (13831932L, 0L): '7', (13832321L, 0L): '6', (13831871L, 0L): '11', (13830166L, 0L): '3', (13830171L, 0L): '6', (13831820L, 0L): '10', (13830755L, 0L): '9', (13831926L, 0L): '5', (13830167L, 0L): '4', (13830172L, 0L): '11', (13830776L, 0L): '8'}
